poster/src/post_with_retries.ts

```diff
@@ -37,8 +37,9 @@ async function postWithRetries(transaction: TransactionConfig, signerKey: string
   console.log(`Posting from account: ${pubKey.address}`);
 
   // TODO: Why manual nonce management?
-  let nonce = await web3.eth.getTransactionCount(pubKey.address)
-  transaction.nonce = nonce
+  // Actually is really unsafe here
```

Comment:
Manual nonce management here was failing for me while sending 9 transactions sequentially for mocking all symbol pairs.

Reply:
there was some bug along the way where nonce was null based on some silly code deep in web3. Maybe it's no longer an issue, but if it comes up again, this may be why

Reply:
yes, I just noticed that actual `postPrices` sometimes fails without this manual nonce managing, and updating uniswap mocked pairs is failing with it. =(

Need some hybrid thing here